### PR TITLE
STYLE: Improve the first interaction message style

### PR DESCRIPTION
**Pull request by jhlegarreta**
Improve the first interaction message style:
- Remove the section header markup (`#`) and avoid an excessively large
  font/avoid making the interaction message prevail over the issue/PR
  text itself.
- Use the YAML literal block style syntax (`|`) and remove the newline
  explicit markup.

## PR Checklist
- [X] No [API changes](https://github.com/InsightSoftwareConsortium/ITK/blob/master/CONTRIBUTING.md#breaking-changes) were made (or the changes have been approved)
- [X] No [major design changes](https://github.com/InsightSoftwareConsortium/ITK/blob/master/CONTRIBUTING.md#design-changes) were made (or the changes have been approved)

### CODE REVIEW

**Comment by jhlegarreta** [comment]
Before:
first_interaction_before

After (intended):
first_interaction_after

**Review by thewtex — APPROVED** [review]
Excellent, thank you for the improvement @jhlegarreta !

File: .github/workflows/first-interaction.yml [hunk 1]
```diff
@@ -15,5 +15,13 @@ jobs:
...
-        pr-message: '# Thank you for contributing a pull request\n**Welcome to the ITK community!**\nWe are glad you are here and appreciate your contribution. Please keep in mind our [community participation guidelines](https://github.com/InsightSoftwareConsortium/ITK/blob/master/CODE_OF_CONDUCT.md).\nMore support and guidance on the contribution process and be found in our [contributing guide](https://github.com/InsightSoftwareConsortium/ITK/blob/master/CONTRIBUTING.md).'
+        issue-message: |
+          Thank you for contributing an issue!
+          **Welcome to the ITK community!**
+          We are glad you are here and appreciate your contribution. Please keep in mind our [community participation guidelines](https://github.com/InsightSoftwareConsortium/ITK/blob/master/CODE_OF_CONDUCT.md).
+          Also, please check [existing open issues](https://github.com/InsightSoftwareConsortium/ITK/issues) and consider discussion on the [ITK Discourse](https://discourse.itk.org).
+        pr-message: |
+          Thank you for contributing a pull request!
```
Comment (thewtex) [inline .github/workflows/first-interaction.yml:24]: a new line here would be nice

File: .github/workflows/first-interaction.yml [hunk 2]
```diff
@@ -15,5 +15,13 @@ jobs:
...
+          **Welcome to the ITK community!**
```
Comment (thewtex) [inline .github/workflows/first-interaction.yml:25]: a new line here would be nice

File: .github/workflows/first-interaction.yml (hunk 1 above)
Comment (jhlegarreta) [reply .github/workflows/first-interaction.yml:26]: Done in a260272.

File: .github/workflows/first-interaction.yml (hunk 2 above)
Comment (jhlegarreta) [reply .github/workflows/first-interaction.yml:28]: Done in a260272.

**Review by dzenanz — APPROVED** [review]

**Review by thewtex — APPROVED** [review]
💯

**Comment by thewtex** [comment]
Thank you @jhlegarreta !

**Pull request merged**